schema/software/template_parameters.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$id": "software/template_parameters.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "template parameters schema",
+    "type": "object",
+    "properties": {
+        "fixedContext": {
+            "type": "object",
+            "description": "fixed parameters for the template"
+        },
+        "requiredKeys": {
+            "type": "object",
+            "description": "keys required to run the flavor"
+        },
+        "conflictingKeys": {
+            "type": "object",
+            "description": "keys that interfere with the flavor and cannot be used"
+
+        }
+    },
+    "required": [
+        "fixedContext",
+        "requiredKeys",
+        "conflictingKeys"
```

Comment:
I don't think we need to make `conflictingKeys` required. Perhaps sometimes there are no conflicting keys.

Reply:
removed